### PR TITLE
add section describing how to install with homebrew

### PR DESCRIPTION
**Pull request by ngoldbaum**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,8 +95,14 @@
 					Python distribution. MacPorts usually includes new releases almost
 					immediately, but Anaconda and Canopy may not always include the latest
 					version.</p>
+                <p>
+                  If you use the <a href="">Homebrew</a> package manager, you
+                  can install astropy by entering the following command in a
+                  terminal:
+                </p>
+                <pre>pip install astropy</pre>
```
Comment (astrofrog) [inline index.html]: pip -> brew? or do you really mean pip?

Comment (ngoldbaum) [inline index.html]: I really mean pip: http://docs.brew.sh/Homebrew-and-Python.html

Homebrew doesn't accept formulae for python libraries that can be pip installed.

Comment (astrofrog) [inline index.html]: I wonder if there's a way to phrase it to make it clear this isn't a typo?

**Comment by ngoldbaum** [comment]
@astrofrog sorry for taking so long to respond, take a look again

**Comment by granttremblay** [comment]
Looks good, @ngoldbaum!

**Pull request merged**